bad_connector_woocommerce/__manifest__.py

```diff
@@ -11,6 +11,7 @@
         "product",
         "sale_management",
         "delivery",
+        "mrp",
```

Comment:
not in original module. you have to create a extension module for the group product